src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -1740,6 +1747,32 @@ int wm_vuldet_send_removed_cve_report (cJSON* j_vuln, scan_ctx_t* scan_ctx) {
         cJSON_AddStringToObject(j_alert_cve, "status", VULN_CVES_STATUS_SOLVED_LOWERCASE);
         cJSON_AddStringToObject(j_alert_cve, "type", cve_type);
         cJSON_AddStringToObject(j_alert_cve, "title", alert_title);
+        cJSON_AddStringToObject(j_alert_cve, "severity", severity);
+        cJSON_AddStringToObject(j_alert_cve, "published", published);
+        cJSON_AddStringToObject(j_alert_cve, "updated", updated);
+
+        // Add CVSS base score
+        if (j_cvss2_score
```

Comment:
Here, we should replicate the corresponding section of `wm_vuldet_send_cve_report()`, because if we only have one of the CVSS scores, none will be present in the alert.

On the other side, if we are comparing **double** variables, we might use `>=` or `<=` instead of `=` and `!=` like we see in `wm_vuldet_fill_report_nvd_scoring()`

Reply:
Done: 7452f0e1874d4c953c66af11a63720226b217440